test/fixtures/dataElement/SpecialWordsElement.txt

```diff
@@ -0,0 +1,13 @@
+Grammar:    DataElement 5.1
+Namespace:  shr.test
+Uses:       shr.core // This shouldn't be needed, but due to bug in importer, it is!
```

Comment:
This is an issue that should be resolved, but is independent of the focus of the PR.

(this comment will be edited with a GH issue link)